t/15-assets.t

```diff
@@ -30,11 +30,12 @@ use Data::Dump qw(pp dd);
 use File::Path qw(remove_tree);
 use File::Spec::Functions 'catfile';
 use Test::More;
-use Test::Warnings;
+use Test::Warnings ':no_end_test';
```

Comment:
That's cheating 😈

Reply:
@dasantiago Any particular reason for this?

Reply:
@foursixnine 

this is the warning that is triggering that test to fail:
"
Archive::Extract will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at lib/OpenQA/Schema/Result/Assets.pm, line 25.
"

Reply:
put Archive::Extract > 0.7 in the cpanfile to get over that warning? 5.18 had 0.68